lib/membrane_live/tokens.ex

```diff
@@ -0,0 +1,38 @@
+defmodule MembraneLive.Tokens do
+  @moduledoc """
+  Module for generating, signing, verifying and validating the tokens used in the app.
+
+  Google Token: - decode:   verification and validation
+  Custom Token:
+    - encode: generation and signing
+    - decode: verification and validation   (#TODO)
```

Comment:
Is it still TODO?

Reply:
Yes, it will be done with [LIVE-72](https://membraneframework.atlassian.net/jira/software/c/projects/LIVE/boards/11?modal=detail&selectedIssue=LIVE-72)